geopandas/base.py

```diff
@@ -540,6 +540,55 @@ def has_z(self):
         """
         return _delegate_property("has_z", self)
 
+    def get_precision(self):
+        """Returns a ``Series`` of the precision of each geometry.
+
+        If a precision has not been previously set, it will be 0 (double precision).
```

Comment:
```suggestion
        If a precision has not been previously set, it will be 0, indicating regular double precision coorindates are in use.
```
I think this is a little clearer (I know the original text comes from shapely)